tests/predictors/sentence_tagger_test.py

```diff
@@ -5,7 +5,7 @@
 
 class TestSentenceTaggerPredictor(AllenNlpTestCase):
     def test_predictions_to_labeled_instances(self):
-        inputs = {"sentence": "Eric Wallace was an intern at AI2"}
```

Comment:
I'm confused but [sequence_tagging.tsv](https://github.com/allenai/allennlp/blob/main/test_fixtures/data/sequence_tagging.tsv) specified in [test_fixtures/simple_tagger/experiment.json](https://github.com/allenai/allennlp/blob/main/test_fixtures/simple_tagger/experiment.json#L3) is not NER task but position-wise classification. In this setting, [predictions_to_labeled_instances](https://github.com/allenai/allennlp/blob/main/allennlp/predictors/sentence_tagger.py#L87-L102) should return empty result, I think. So, I retrained the model, update the snapshot, and update the test. (If I have some misunderstanding, please correct me.)